plume-models/src/instance.rs

```diff
@@ -138,7 +130,7 @@ impl Instance {
         open_registrations: bool,
         short_description: SafeString,
         long_description: SafeString,
-    ) {
+    ) -> Result<usize> {
```

Comment:
If I'm not mistaken, the usize is going to be how many entries were affected, which is one?

Reply:
Again, https://github.com/Plume-org/Plume/pull/392#discussion_r244369771